package.json

```diff
@@ -7,6 +7,7 @@
   "sideEffects": false,
   "scripts": {
     "prepare": "npm run clean && npm run build:cjs && npm run build:esm",
+    "prettier": "prettier \"{src,test}/**/*.js\"",
```

Comment:
I've used lint-staged for prettier in other projects. This is fine as-is but might be worth adding that as a follow up

Reply:
Yeah I'll open a new PR for that once this has been merged.